src/main/scripts/org/reaktivity/specification/nukleus/oauth/streams/proxy/request.with.too.many.scopes.with.signed.jwt.rs256.forwarded/accept/client.rpt

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property oauthToken ${oauth:jwt("RS256")
+                           .claim("iss", "test issuer")
+                           .claim("scopes", "1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49toomany")
```

Comment:
The concept of too many scopes should be tested via a controller script with `RESOLVE` attempting to resolve more than `48` scopes.

Presenting a request token with `100` scopes would not be a problem because only scopes that were previously `RESOLVED` would be included in the resulting `authorization` bitmask. Given the enforcement described directly above, that will be limited to `48` or fewer.

Recommend changing this test to send just `3` scopes, as follows; `role1`, `role3`, and `role4` and verify that the request is _not_ rejected, but forwarded with authorization bits set correctly for `role1` and `role3`, i.e. `0x0001_000000000005L`.